### ISSUE TITLE
Scale duration slider to account for content min and max times

### ISSUE THREAD

**Issue by augustf** [concerto/concerto#1121]
Right now, the duration slider is optimized for our default content duration times. Users setting longer times have pointed out its almost impossible to get the values they want on the scale provided. Perhaps making the slider responsive to free-input or keyboard commands would help.

**Comment by echathaz** [comment]
I do wish there were a way to set the time via a text field or possibly additional buttons to increase or decrease as a single integer. This was an issue for me when I was showing live streams from YouTube as an iframe since the live streams throw an error when attempting to add via the video submission since there is no end time...

**Comment by Baylink-gh** [comment]
Yeah, that was my question too: What's the defined solution to "my content is a live stream that's gonna be almost 2 hours, and I don't know how long"?
